src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -193,6 +205,18 @@ public abstract class AbstractExplorerPresenter<TView, TPresenter> : AbstractPre
 
       protected abstract void AddProjectToTree(IProject project);
 
+      private void addClassifiableUnderActiveClassification(IClassifiable classifiable)
+      {
+         if ((_activeClassification?.ClassificationType == classifiable.ClassificationType) && classifiable.Parent == null)
```

Comment:
if there is an active classification that was set (for instance via context menu click) then we are going to set the parent of the classifiable to this classification

And then we reset to be sure :)